css/style.css

```diff
@@ -0,0 +1,84 @@
+/**
+ * Place your custom styles here.
+ */
+
+ /*GLOBAL OVERRIDES*/
+@import url('https://fonts.googleapis.com/css?family=Raleway');
+@import url('https://fonts.googleapis.com/css?family=Oswald');
+p,li,dt,dd,td,th,details,button,form,a{font-family: Raleway;}
+.main-container{margin-top: 5px;}
+img {max-width: 100%; height: auto;}
+
+/*top menu for subsites*/
+.subsites-entry{}
+.subsites-entry li.collections{background-color:#FF4519;}
+.subsites-entry li.shop{background-color:#6EB209;}
+
+/*side menu or "utilities menu"*/
+.region-sidebar-second .nav li{border-bottom:1px dotted #ddd;}
+.region-sidebar-second .nav li:last-child{border-bottom:none;margin-bottom:15px;}
+.region-sidebar-second .nav li a{color:#777;}
+
+/*search box*/
+.region-sidebar-second .btn-primary {
+    background-color: #0f245c;
+    border-color: #0f245c;
+}
+
+/*heading for the featured image*/
+.region-sidebar-second .view-header .panel-body{padding:0;}
+.region-sidebar-second .view-header h2{color:#777;font-size:1.2em;margin-top:5px;margin-bottom:5px;}
+
+
+/* ANY AREA WITH BG COLOUR WILL BE OVERRIDDEN WITH LESS FORMAT COMING LATER*/
+/* navbar for top menu */
+.navbar-nav{background-color: #0f245c;}
+.navbar-inverse {background-color: #0f245c;border-color: #0f245c;}
+.navbar-nav li a{font-family: "News Cycle", "Arial Narrow Bold", sans-serif;font-weight:normal;}
+
+/* START UNSUSTAINABLE HACKS*/
+
+/*slider overrides - note this is a hack, only works on fixed sizes
+.form-group .view-content {max-height: 290px}
+.form-group .view-content h2{display:none;}
+
+/*END HACKS*/
+
+/*content pages image management*/
+.image-left-float-panel{float:right;margin:10px 0px 10px 20px;}
+figcaption{font-style: italic;}
+.panel .center{margin: auto; text-align:center;}
+
+/*hacks on the drop down menus to make them appear like another site - note only works in desktop and produced for the wireframes
```

Comment:
If this stuff is only temporary and you don't want it any more, then remove it rather than commenting it out.